.github/SUPPORT.md

```diff
@@ -0,0 +1,32 @@
+Support Guide
+===
+
+Welcome to the support guide! What type of help are you looking for?
+
+## I need help understanding how the app works
+If you want to learn how use a specific feature of the application, please see our [user guide](https://help.openboxes.com/), which describes the primary features of the app in non-technical terms.
+
+## I need help running/deploying the app
+Have you read through our [installation documentation](http://docs.openboxes.com/en/latest/installation/)? The documentation outlines all the steps that a system administrator should need to follow to install, configure, update, and deploy the application to a production server.
+
+If you're running into an issue not described in our documentation, take a look through the [support forums](https://openboxes.uservoice.com/forums/163294-general) to see if someone has encountered a similar problem. If you're still stuck, we encourage you to create a new forum post. Please make sure to let us know the step of the installation process that you're stuck on.
```

Comment:
I think the "support forum" should link to Github Issues and community.openboxes.com instead of the UserVoice forum.